### PR TITLE
 fix language select

### PR DESCRIPTION
**Pull request by mateuszpmroz**

### CODE REVIEW

File: src/navigators/Navigator.tsx [hunk 1]
```diff
@@ -120,6 +125,7 @@ const mapStateToProps = (state: ApplicationState): MapStateToProps => ({
   isPinSet: selectors.isPinSet(state),
   isTxPasswordSet: selectors.isTxPasswordSet(state),
   isAuthenticated: selectors.isAuthenticated(state),
+  language: state.appSettings.language,
```
Comment (KedziaPawel) [inline src/navigators/Navigator.tsx:128]: selector for this

File: src/screens/Settings/SelectLanguageScreen.tsx [hunk 2]
```diff
@@ -41,8 +42,10 @@ const LanguageItem = ({ language, selectedLanguageValue, onLanguageSelect }: Lan
 };
 
 export const SelectLanguageScreen = (props: SelectLanguageScreenProps) => {
-  const [isLoading, setIsLoading] = useState(true);
-  const [selectedLanguageValue, setselectedLanguageValue] = useState('en');
+  const { language } = useSelector((state: ApplicationState) => ({
+    language: state.appSettings.language,
```
Comment (KedziaPawel) [inline src/screens/Settings/SelectLanguageScreen.tsx:46]: create a selector for this and put it into useSelector

File: src/screens/Settings/SelectLanguageScreen.tsx [hunk 3]
```diff
@@ -85,20 +79,16 @@ export const SelectLanguageScreen = (props: SelectLanguageScreenProps) => {
...
     <ScreenTemplate
       header={<Header isBackArrow={true} navigation={props.navigation} title={i18n.selectLanguage.header} />}
     >
-      {availableLanguages.map(language => (
+      {availableLanguages.map(item => (
         <LanguageItem
-          language={language}
-          selectedLanguageValue={selectedLanguageValue}
+          selectedLanguage={item}
+          selectedLanguageValue={language || 'en'}
```
Comment (KedziaPawel) [inline src/screens/Settings/SelectLanguageScreen.tsx:89]: the default value is in redux just `selectedLanguageValue={language}`

**Comment by KedziaPawel** [comment]
I see that you didn't reinstall pods, make sure that the app is working on ios

**Comment by mateuszpmroz** [comment]
> I see that you didn't reinstall pods, make sure that the app is working on ios

I'm working on Ubuntu so I can't run xcode to simulate ios

**Comment by mateuszpmroz** [comment]
@KedziaPawel @Kamill90 Please check the PR again if possible.

File: src/screens/Settings/SelectLanguageScreen.tsx [hunk 4]
```diff
@@ -41,8 +43,11 @@ const LanguageItem = ({ language, selectedLanguageValue, onLanguageSelect }: Lan
 };
 
 export const SelectLanguageScreen = (props: SelectLanguageScreenProps) => {
-  const [isLoading, setIsLoading] = useState(true);
-  const [selectedLanguageValue, setselectedLanguageValue] = useState('en');
+  const { language } = useSelector((state: ApplicationState) => ({
```
Comment (KedziaPawel) [inline src/screens/Settings/SelectLanguageScreen.tsx:46]: still missing selector

**Pull request merged**